tests/value/test_semivalues.py

```diff
@@ -52,15 +54,29 @@ def test_marginal_batch_size(test_game, sampler, coefficient, batch_size, seed):
     marginals_single = []
     for sample in samples:
         marginals_single.extend(
-            _marginal(test_game.u, coefficient=coefficient, samples=[sample])
+            DefaultMarginal()(test_game.u, coefficient=coefficient, samples=[sample])
         )
 
-    marginals_batch = _marginal(test_game.u, coefficient=coefficient, samples=samples)
+    marginals_batch = DefaultMarginal()(
+        test_game.u, coefficient=coefficient, samples=samples
+    )
 
     assert len(marginals_single) == len(marginals_batch)
     assert set(marginals_single) == set(marginals_batch)
 
 
+@pytest.mark.parametrize("num_samples", [5])
+def test_msr_banzhaf(
+    num_samples: int, analytic_banzhaf, parallel_config, n_jobs, seed: Seed
+):
+    u, exact_values = analytic_banzhaf
+    values = compute_msr_banzhaf_semivalues(
+        u=u, done=MaxChecks(1000), config=parallel_config, n_jobs=n_jobs, seed=seed
+    )
+    # Need to use atol because msr banzhaf is quite noisy.
+    check_values(values, exact_values, atol=0.1)
```

Comment:
**Suggestion** Perhaps it would make sense to all make sure that the rankings are the same.